core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/molecule/input/CheckboxInput.kt

```diff
@@ -29,7 +42,11 @@ fun CheckboxInput(
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .clickable { onCheckedChange(!checked) },
+                .clickable { onCheckedChange(!checked) }
+                .semantics(mergeDescendants = true) {}
+                .clearAndSetSemantics {
+                    contentDescription = combinedContentDescription
+                },
```

Comment:
Looks like we could use `.toggleable(value = checked) { onCheckedChange(!checked) }` instead.

See https://stackoverflow.com/a/78059984

We should also use a `Checkbox` with `onCheckedChange` set to `null` so the checkbox is not individually selectable using Talkback.

Reply:
Looks like this approach is successful, and if I use mergeDescendents it will also work well without a combined description.

Reply:
Hmm with this approach talkback reads "checked, detected icon, checkmark, use compression, detected icon, checkmark, checkbox."

I don't know where it is getting icon from, and why it is repeating checkmark/checkbox so often. I guess I'd need to clear semantics somewhere. 

I'll upload the new version in a bit as an intermediate.

Reply:
I'm going to categorize this as "better than before", we can do more work here with some targeted a11y work.